packages/plugin-ext/src/plugin/known-commands.ts

```diff
@@ -63,6 +65,9 @@ export namespace KnownCommands {
         }
     };
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const ID = (args: any[]) => args;
```

Comment:
Minor: This is supposed to be an identity function, right? I would usually associate `ID` with an `Identifier`, not an `Identity`.
```suggestion
    const identity = (args: any[]) => args;
```